latest/Dockerfile

```diff
@@ -39,10 +39,10 @@ COPY root/usr/libexec/fix-permissions /usr/libexec/fix-permissions
 # This image must forever use UID 26 for postgres user so our volumes are
 # safe in the future. This should *never* change, the last test is there
 # to make sure of that.
-RUN yum install -y centos-release-scl-rh && \
+RUN yum install -y centos-release-scl && \
```

Comment:
What are the differences between these two repositories? @hhorak

Reply:
`centos-release-scl` allows also community developed collections and packages - `sclo-ror42`, `sclo-subversion19`, `sclo-vagrant1`,...

Reply:
right, and I think we should stay with `centos-release-scl-rh`, to be sure we don't install anything from the other set..